### ISSUE TITLE
数据集划分问题

### ISSUE THREAD

**Issue by hy-pn** [ggsDing/Bi-SRNet#10]
在将数据集划分为训练集以及测试集的时候，验证集占比多少，是按数据集的顺序划分的还是随机抽取划分的

**Comment by hy-pn** [comment]
或者能不能麻烦您将您训练时的数据training set,val set, test set分享一下？

**Comment by ggsDing** [comment]
> 或者能不能麻烦您将您训练时的数据training set,val set, test set分享一下？

可以的，过两天

**Comment by hy-pn** [comment]
不知道是不是您忘记了，所以再一次打扰您，想问一下现在可以分享一下您的数据吗



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2022年7月26日(周二) 下午2:03
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [ggsDing/Bi-SRNet] 数据集划分问题 (Issue #10)




  
或者能不能麻烦您将您训练时的数据training set,val set, test set分享一下？
  
可以的，过两天

**Comment by hy-pn** [comment]
我进行了随机抽取，训练、验证、测试集7：1：2，但测试结果还是达不到您论文里的精度

**Comment by ggsDing** [comment]
Hi. I have uploaded the lists of training and validation images in the 

> /dataset

 directory, please check.
However, I have not run the experiments for quite some time, so I am not sure if this is the train/val split in the published paper.

**Comment by hy-pn** [comment]
您的论文里的那些评价指标在哪个（validation/test）集上计算的？您是使用什么数据进行测试（test）的呢，除您的training set与validation set没有其他的公开数据了啊

**Comment by hy-pn** [comment]
按照您的代码里的内容，数据集应该有三部分，应该还有测试集，不知道您的测试集是哪些图像呢

**Comment by ggsDing** [comment]
> 按照您的代码里的内容，数据集应该有三部分，应该还有测试集，不知道您的测试集是哪些图像呢

During the contest, there was a testing set that can be evaluated on the test server. However, the released SECOND dataset does not provide labels for the testing set. So the evaluation was conducted on the validation set in the paper.

See discussion in the paper:
> Among the 4662 pairs of temporal images, 2968 ones are openly available. We further split them into a training set and a test set with the numeric ratio of 4 : 1 (i.e., 2375 image pairs for training and 593 ones for testing).

**Comment by hy-pn** [comment]
好的，谢谢您的回答，感谢

**Comment by linyiyuan11** [comment]
> 在将数据集划分为训练集以及测试集的时候，验证集占比多少，是按数据集的顺序划分的还是随机抽取划分的

您好，所以您知道是顺序划分还是随机抽取的吗

**Comment by ggsDing** [comment]
> 

Please check the detailed split of train/test sets in the '/datasets' folder
